### DISCUSSION TITLE
Condensation.. how to solve?

### DISCUSSION THREAD

**Discussion by spanzetta** [jomjol/AI-on-the-edge-device#3118] [General] [answered]
I am having a big problem with condensation on the meter glass now that outside temperature are raising (this winter I didn't have this problem).. due to the cold water temperature..
What happens is that the glass is totally wet.. and it becomes of course hard to recognize the numbers..
See image below as example (sometimes it's even worst)
After I manually dry the glass it works perfectly.. but then again the problem of condensation starts (now after few hours)
Did someone experience the same.. and how did you solve?
Thanks

<img width="316" alt="image" src="https://github.com/jomjol/AI-on-the-edge-device/assets/49818743/aade1362-55aa-4cc5-bc65-6e11cb3b20ae">

**Comment by friedpa** [comment, marked as answer]
I used a spray, which you can buy in a car equipment shop. I think it´s called anti-fogging spray in English

**Reply by spanzetta** [reply]
And this should fix the condensation problem caused by the different temperature ??
It would be great.. will try

**Reply by friedpa** [reply]
It worked in my case....
